### PR TITLE
Hxntools at srx and interpolation function

### PR DESCRIPTION
**Pull request by licode**

### CODE REVIEW

File: pyxrf/model/fileio.py [hunk 1]
```diff
@@ -65,11 +65,15 @@
     # registers a filestore handler for the XSPRESS3 detector
     from hxntools import handlers as hxn_handlers
 except ImportError as e:
-    logger.error('hxntools is not available: %s', e)
+    logger.error('hxntools is not available from old version: %s', e)
 
+try:
+    import vortex_handler
```
Comment (klauer) [inline pyxrf/model/fileio.py]: Where does `vortex_handler` come from?

File: pyxrf/model/fileio.py [hunk 2]
```diff
@@ -65,11 +65,15 @@
...
+except ImportError as e:
+    logger.error('handler is not loaded.')
 
 
-beamline_name = 'HXN'  # this name will be changed according to beam lines.
+beamline_name = 'SRX'  # this name will be changed according to beam lines.
```
Comment (klauer) [inline pyxrf/model/fileio.py]: Is it OK to hard-code this?

File: pyxrf/model/fileio.py (hunk 1 above)
Comment (licode) [inline pyxrf/model/fileio.py]: This is a temporary fix at SRX. @tacaswell copied some handler from hxntools and created a new repository for that. I think the long term plan is to move the handlers into filestore?

File: pyxrf/model/fileio.py (hunk 2 above)
Comment (tacaswell) [inline pyxrf/model/fileio.py]: No

Comment (licode) [inline pyxrf/model/fileio.py]: Good catch. beamline_name will not be used when users play with pyxrf in their laptops, this name is only used when pyxrf talks to databroker. I know this is not optimal, and my plan it to put the option at GUI level.

**Pull request merged**